lib/classes/class-bootstrap.php

```diff
@@ -1998,6 +2051,49 @@ function get_image_sizes($unset_disabled = true) {
 
         return $sizes;
       }
+
+      /**
+       * Get the email for notifications
+       *
+       * @return string
+       */
+      public function get_notification_email() {
+        $type = $this->get( 'sm.status_email_type' );
+
+        switch ($type) {
+          case 'true':
+            // Emails to Admin 
+            return get_option('admin_email');
```

Comment:
@balexey88 Do we have this field in Network settings? 
I think it can be set for all Sites in the Network, and in this case we have to check this value as well using `get_site_option`

Reply:
Yes, thank you!
Fixed https://github.com/udx/wp-stateless/commit/e93b7550742f3e31acc6e30449340e30b70588b0